pybamm/meshes/zero_dimensional_submesh.py

```diff
@@ -33,3 +35,32 @@ def __init__(self, position, npts=None):
     def add_ghost_meshes(self):
         # No ghost meshes to be added to this class
         pass
+
+
+class MeshGenerator0D(MeshGenerator):
+    """
+    A class to generate a submesh on a 1D domain.
+
+    Parameters
+    ----------
+
+    submesh_type: str, optional
+        The type of submeshes to use. Can be "Position". Default is "Position".
```

Comment:
Maybe a little more documentation on what is meant by "Position" here. Do we just mean a spatial mesh? or something else